app/components/Reactions/Reaction.tsx

```diff
@@ -91,28 +91,34 @@ const Reaction = ({
     return <></>;
   }
 
-  let tooltipContent =
-    showPeople && optimisticUsers?.map((user) => user.fullName).join(', ');
-  tooltipContent = tooltipContent ? `${tooltipContent} reagerte med ` : '';
-  tooltipContent += emoji;
+  const tooltipContentNames =
+    showPeople &&
+    optimisticUsers
+      ?.map((user) => user.fullName)
+      .sort()
+      .map((name) => <div key={name}>{name}</div>);
+  const tooltipContent = tooltipContentNames ? (
+    <div>
+      {tooltipContentNames}
+      har reagert med {emoji}
+    </div>
+  ) : undefined;
```

Comment:
Why the added divs etc? Did the current way not work?

Reply:
I think it's to get them on separate lines. Now all the names are all on one long line, and it's impossible to read them

Reply:
Originally I made a component to take care of the information in tooltips. The component became redundant. Now the main reason the divs are there , as Eik said, are to make it cleaner and separate each name on a new line. There may be better ways to do that.

Reply:
Can you use a shard (<></>) with display: block, or just insert newline characters (\n)?

Reply:
`showPeople && optimisticUsers?.map((user) => user.fullName).join(', \n');`
Something like this?

Reply:
Shards may work ( I am not sure if shards are inline or blocks). My original plan was, as you said, to use \n, but it gets interpreted as HTML or something and and newlines just get switched out with spaces (I may have done it wrong).

Reply:
Turns out the shards are also inline

Reply:
Shards are not rendered as html, so it will work exactly the same as just having the names directly after each other

Reply:
But adding "\n" after each line seems like a good solution imo.

Reply:
My original plan was to use \n, but it gets interpreted as HTML or something and newlines just get switched out with spaces (I may have done it wrong). How should I use implement newlines?